pkg/datapath/loader/netlink.go

```diff
@@ -506,7 +507,7 @@ func setupTunnelDevice(mode string, port, mtu int) error {
 //
 // Changing the destination port will recreate the device. Changing the MTU will
 // modify the device without recreating it.
-func setupGeneveDevice(dport, mtu int) error {
+func setupGeneveDevice(dport uint16, mtu int) error {
```

Comment:
Should we dry this up? ie setupDevice(mode, port, mtu) or use generics?? likely a separate PR...

Reply:
Yeah, I'd leave that out of this PR to avoid extending too much its scope.